libs/platform/experience/src/services/layout/plugins/types/collapsible/collapsible.plugin.ts

```diff
@@ -27,10 +27,12 @@ export class CollapsibleLayoutPlugin implements LayoutPlugin {
     if (data.isComposition) return of();
 
     return of({
-      outer: html`<oryx-collapsible ?open=${data?.options?.collapsibleOpen}
-        >${renderLabelSlot(data, 'heading', -1)}
-        ${data.template}</oryx-collapsible
-      >`,
+      outer: html`<oryx-collapsible
+        ?open=${data?.options?.collapsibleOpen}
+        .syncKey=${data.options.collapsibleSyncKey}
```

Comment:
The same, what about
```suggestion
        .stateKey=${data.options.collapsibleSyncKey}
```
or
```suggestion
        .persistatedStateKey=${data.options.collapsibleSyncKey}
```